Makefile

```diff
@@ -35,17 +42,15 @@ install_simplified:
 	fi
 	make _initiate_env
 	curl -LsSf https://astral.sh/uv/install.sh | sh
-	uv || echo -e "\n \n Please restart you shell in order to allow uv commands \n \n"
-	uv venv --python 3.10
-	source .venv/bin/activate && uv pip install -r requirements.txt && pre-commit install
+	uv venv --python 3.10  # Create a virtual environment using uv
```

Comment:
je suis chaud que tu laisses la ligne 
`uv || echo -e "\n \n Please restart you shell in order to allow uv commands \n \n"`(qui indique quoi faire si jamais uv n'est pas installé)